picard/ui/itemviews.py

```diff
@@ -859,6 +862,7 @@ def __init__(self, obj, sortable, *args):
         self.setTextAlignment(1, QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter)
         self.setTextAlignment(7, QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter)
         self.setTextAlignment(8, QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter)
+        self.setSizeHint(MainPanel.TITLE_COLUMN, ICON_SIZE)
```

Comment:
How does this related to current patch?

Reply:
Technically this thing is the key to responsiveness and performance. 35% of CPU time was wasted guessing the icon size during each file update. This single line prevents that.
https://github.com/metabrainz/picard/pull/1555#issuecomment-644475419

Reply:
Is there anywhere else in Picard where we need to do this?

Reply:
Nope. Just a single line. The hinted size is used in all updates of the items.

Other place that called for size hints was the rowheight to determine the vertical scrollbar size. Mitigated by setting uniform row height.

The last place that can use this are the horizontal scrollbars, that try to guess the width based on column contents. Way harder to mitigate that.

Reply:
On the _clone side of things (QtThreads), most of the time is spent calculating the coverart hash (10% of CPU time) and decoding the file metadata (22% of CPU time).
![perf3](https://user-images.githubusercontent.com/1678454/84844986-022b7300-b022-11ea-9113-72ba9e5d4613.png)

Reply:
Tried replacing the md5 with a fletcher variant (Addler32), the hash column is gone but didn't seem to affect much. Ubuntu seems to be bottlenecked by the disk drivers (ntfs-3g). Tried loading from a sata SSD on Windows, but nothing changed. 

I assume the bottleneck has moved not to the CPU time, nor to the disk, but wait due to syscalls.